io.github.LogoShareHub.LogoShareHub.yaml

```diff
@@ -0,0 +1,16 @@
+id: io.github.LogoShareHub.LogoShareHub
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --share=network
+command: lsh-search
+modules:
+  - name: search
+    buildsystem: simple
+    build-commands:
+      - install -D scripts/lsh-search /app/bin/lsh-search
+    sources:
+      - type: git
+        url: "https://github.com/LogoShareHub/LogoShareHub.git"
+        branch: "main"
```

Comment:
Don't use a branch, must be a commit or tag

Reply:
what language i use the cli in